static/app/components/onboardingWizard/newSidebar.tsx

```diff
@@ -39,6 +39,11 @@ import {useLocalStorageState} from 'sentry/utils/useLocalStorageState';
 import useOrganization from 'sentry/utils/useOrganization';
 import useRouter from 'sentry/utils/useRouter';
 
+/**
+ * How long (in ms) to delay before beginning to mark tasks complete
+ */
+const INITIAL_MARK_COMPLETE_TIMEOUT = 600;
```

Comment:
nit: Should we maybe make this smaller, it seems pretty slow in the video you have shared, i am not sure if users will wait and keep attention that long at the tasks list?

Reply:
yes the same feedback I got from Evan... I will set `450` and see if it improves

Reply:
I think the speed of the animation itself also feels slow. Like the actual motion. I would expect them to switch around a bit snappier. We had animation on these before right?

Reply:
yes we did have but I find the animation not very smooth/natural. do you prefer that?

https://github.com/user-attachments/assets/c9d288cc-8535-4d63-82c5-e1a1df9d8d78

Reply:
I actually think this feels better. Snappier animation generally makes the app feel a little bit "faster" if that makes sense